src/Cell/index.tsx

```diff
@@ -138,12 +138,9 @@ function Cell<RecordType extends DefaultRecordType>(
   const isFixRight = typeof fixRight === 'number';
 
   if (isFixLeft) {
-    fixedStyle.position = 'sticky';
```

Comment:
@zombieJ We need `-webkit-sticky` to fix issue in Safari 12.

Reply:
We need to set `-webkit-sticky` and sticky at the same time and let the browser choose which value to use.

In addition, because there can be no duplicate keys in the object, it is written in its corresponding calss;

Reply:
We also need `width: 30px` to fix issue in Safair 12. 
You can see this closed [issue](https://github.com/ant-design/ant-design/issues/22773).
This issue has been resolved in AntDesign, but not in react-component / table.

Reply:
should still keep this since it will break current version with patch.

Reply:
Let me add antd less side directly.

Reply:
How about https://www.npmjs.com/package/css-vendor

Reply:
I considered to obtain the current browser prefix by introducing a library or a tool function, but for careful consideration, only the CSS was modified.